services/app-api/serverless.yml

```diff
@@ -1,3 +1,8 @@
+## set the values for the following ssm parameters in AWS SSM Parameter store for app-api service. You will need to set the values and also a default value to fall to if the required value is not present.
+# 1. iamPath:  /configuration/${self:custom.stage}/iam/path    &  /configuration/default/iam/path
```

Comment:
@joeaero19  a few things
can you document what the purpose of each ssm paramater is ... the "what" it is and the "why" we're using it. 
Another thing is we should explain what self:custom.stage is ... I know what it is , I know the pattern in which we use that by specifying the branch name but that may not be obvious to a consumer of the quickstart. We should explain that pattern

Reply:
also we should note which ssm param is required or optional